kotsadm/operator/kustomize/base/deployment.yaml

```diff
@@ -25,7 +25,7 @@ spec:
               memory: 500Mi
           env:
             - name: KOTSADM_API_ENDPOINT
-              value: http://kotsadm-api-node:3000
+              value: http://kotsadm:3000
```

Comment:
can this be changed before we move everything?  shouldn't there be 2 endpoints now?

Reply:
the operator does not talk to the old api directly anymore